ddl/ddl_api.go

```diff
@@ -4891,6 +4892,93 @@ func (d *ddl) DropIndex(ctx sessionctx.Context, ti ast.Ident, indexName model.CI
 	return errors.Trace(err)
 }
 
+func (d *ddl) DropIndexes(ctx sessionctx.Context, ti ast.Ident, specs []*ast.AlterTableSpec) error {
+	is := d.infoHandle.Get()
+	schema, ok := is.SchemaByName(ti.Schema)
+	if !ok {
+		return errors.Trace(infoschema.ErrDatabaseNotExists)
+	}
+	t, err := is.TableByName(ti.Schema, ti.Name)
+	if err != nil {
+		return errors.Trace(infoschema.ErrTableNotExists.GenWithStackByArgs(ti.Schema, ti.Name))
+	}
+
+	allIndexIsExists := true
+	indexNames := make([]interface{}, 0, len(specs))
```

Comment:
`indexNames := make([]model.CIStr, 0, len(specs))`

Reply:
3q

Reply:
> `indexNames := make([]model.CIStr, 0, len(specs))`

It was fixed